### PR TITLE
Define cid v1 using base32

### PR DESCRIPTION
**Pull request by SimonLab**
ref: #36 
Create a cid v1 with base32.
You can specify which base you want to use in your config
`config :excid, base: :base32`
or
`config :excid, base: :base58`

By default if the config is not defined then the base32 will be used

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,7 +26,14 @@ end
 
 and run `mix deps.get`
 
-2. Call the `cid/1` function with a string, map, or struct as a parameter...
+2. Define in your mix configuration which base you want to use to create the cid.
+The value of the base should be the either `:base32` or `:base58`.
+If the base is not defined then `excid` will use the `base32` by default.
+
+```elixir
+config :excid, base: :base32
```
Comment (RobStallion) [inline README.md:34]: 👍

**Review by Danwhy — APPROVED** [review]
Looks good to me :+1:

File: test/cid_test.exs [hunk 2]
```diff
@@ -106,6 +123,15 @@ defmodule CidTest do
     File.rm!(@filename)
   end
 
+  def compare_ipfs_cid_base32(val) do
```
Comment (RobStallion) [inline test/cid_test.exs:126]: This was really clever 👍

**Review by RobStallion — APPROVED** [review]
LGTM. Nice one 👍

**Review by nelsonic — APPROVED** [review]
@SimonLab nice work! 🥇

**Pull request merged**